packages/deployment-service/serverless.yml

```diff
@@ -375,21 +375,21 @@ functions:
               - Authorization
               - X-Api-Key
               - api-version
-  pipelineRun:
-    handler: src/index.pipelineRun
-    events:
-      - http:
-          path: /pipeline/{id}/run
-          method: post
-          cors:
-            origin: '*'
-            headers:
-              - Content-Type
-              - Authorization
-              - X-Api-Key
-              - api-version
-          authorizer:
-            arn: arn:aws:cognito-idp:${self:provider.region}:${self:custom.env.AWS_ACCOUNT_ID}:userpool/${self:custom.env.CONNECT_USER_POOL}
+  # pipelineRun:
```

Comment:
Should this one be commented out?

Reply:
I kinda still want it for a bit but it will be deleted